### ISSUE TITLE
Schema also Shows for Category and Tag Pages in WordPress

### ISSUE THREAD

**Issue by mj00712** [gehwissenlos/schemaorg-custom-field#1]
Hi. Hope you are doing well. I have used this method to show Rich Snippet on Google for Job Posting. It has worked but Google has placed a manual action on my website for showing Schema on category pages. How can it be used in a way that the Schema only shows on that post only not on category/tag or other pages?
![Capture](https://user-images.githubusercontent.com/44098609/66105132-ff900700-e5d3-11e9-93d9-139626f8186a.JPG)
Thanks

**Comment by gehwissenlos** [comment]
Hi,
could you change the line
`if(!empty($schema))`
to
`if(!empty($schema) && !is_page() && is_single() && is_singular()) {`

**Comment by mj00712** [comment]
THAT HAS FIXED THE ISSUE. 
I want to thank you from deep of my heart. This means a lot to us. I have been searching for this solution for almost 6 months. I found this code on some website and using this code for almost 9 months. This has really helped us a lot. But after the manual action, I couldn't find a solution on any website. I have commented on several websites but no one replied. I have contacted several websites support team but they don't have the solution even some Schema plugin developers. This plugin is a very powerful tool and it is the best. Better than many so-called Top-rated plugins. Thank you again. May your family leave in Peace and Prosperity. Thanks
